### ISSUE TITLE
id->class is not consistently defined

### ISSUE THREAD

**Issue by HPCguy** [jserv/amacc#72]
id->class is 0 for passed parameters vs. class 'Par' or 'Loc'.  There may be other inconsistencies for other storage types.

**Comment by jserv** [comment]
> id->class is 0 for passed parameters vs. class 'Par' or 'Loc'. There may be other inconsistencies for other storage types.

Can you make a list of proposed changes to improve the consistency?

**Comment by HPCguy** [comment]
Yes.  I may pick up this ticket in the next few weeks.  There is a lot of detail work to make everything consistent.  It probably won't be a lot of code changes, but I (or whoever takes this on) will have to check every corner of the code to make sure they have it right.
